query_service/core/routers/rapid_release.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# DISCLAIMER: This software is provided "as is" without any warranty,
+# express or implied, including but not limited to the warranties of
+# merchantability, fitness for a particular purpose, and non-infringement.
+#
+# In no event shall the authors or copyright holders be liable for any
+# claim, damages, or other liability, whether in an action of contract,
+# tort, or otherwise, arising from, out of, or in connection with the
+# software or the use or other dealings in the software.
+# -----------------------------------------------------------------------------
+
+# @Author  : Tek Raj Chhetri
+# @Email   : tekraj@mit.edu
+# @Web     : https://tekrajchhetri.com/
+# @File    : rapid_release.py
+# @Software: PyCharm
+
+from fastapi import APIRouter, Request, HTTPException, status
+from core.graph_database_connection_manager import (fetch_data_gdb, concurrent_query,  convert_to_turtle, insert_data_gdb)
+import json
+import logging
+from core.configuration import load_environment
+from core.shared import (read_yaml_config, yaml_config_list_to_query_dict, yaml_config_single_dict_to_query,
+                         transform_data_categories, clean_response_statistics, clean_response_concatenated_predicate_object)
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+@router.get("/statistics", summary="Statistics",
+            description="This endpoint gets the statistics, i.e., counts, about the rapid release data, e.g., donors sample count.")
+async def get_statistics():
+    file = load_environment()["RAPID_RELEASE_FILE"]
```

Comment:
Do you need to call this upon every hit of the endpoint?

Reply:
I don't want to load it on the startup, and that was the reason for that.